src/openvic-simulation/map/Map.cpp

```diff
@@ -504,6 +403,134 @@ bool Map::load_region_file(ast::NodeCPtr root) {
 	return ret;
 }
 
+static constexpr colour_t colour_at(uint8_t const* colour_data, int32_t idx) {
+	idx *= 3;
+	return (colour_data[idx + 2] << 16) | (colour_data[idx + 1] << 8) | colour_data[idx];
+}
+
```

Comment:
Please comment these numbers in use. Particularly idx *= 3;
It's not clear of the why, though I can see an 8 and 16 division of data. 24 bit color? I don't know.